cockatrice/src/abstractcarddragitem.cpp

```diff
@@ -1,41 +1,52 @@
 #include "abstractcarddragitem.h"
 #include "carddatabase.h"
 #include <QCursor>
-#include <QGraphicsSceneMouseEvent>
 #include <QDebug>
+#include <QGraphicsSceneMouseEvent>
 #include <QPainter>
 
 static const float CARD_WIDTH_HALF = CARD_WIDTH / 2;
 static const float CARD_HEIGHT_HALF = CARD_HEIGHT / 2;
 const QColor GHOST_MASK = QColor(255, 255, 255, 50);
 
-AbstractCardDragItem::AbstractCardDragItem(AbstractCardItem *_item, const QPointF &_hotSpot, AbstractCardDragItem *parentDrag)
+AbstractCardDragItem::AbstractCardDragItem(AbstractCardItem *_item,
+                                           const QPointF &_hotSpot,
+                                           AbstractCardDragItem *parentDrag)
     : QGraphicsItem(), item(_item), hotSpot(_hotSpot)
 {
-    if (parentDrag) {
+    if(parentDrag)
```

Comment:
I'd prefer the spaces after `if` etc be kept.

Reply:
Agreed. Spaces after any conditional word (if/else/for/while/...)

Reply:
Agreed